### PR TITLE
WorkflowGuidelines: common bad practices

### PR DESCRIPTION
**Pull request by aarani**
This commit descibes some bad practices like DRY, Magic Numbers and Primitive Obsession.

### CODE REVIEW

File: WorkflowGuidelines.md [hunk 1]
```diff
@@ -75,3 +75,80 @@ closed after the commit lands, then you would use the word `Closes` instead of
 Do not use long lines (manually crop them with EOLs because git doesn't do this
 automatically).
 
+* Avoid typical bad practices like:
+   * Magic numbers:
+
```
Comment (knocte) [inline WorkflowGuidelines.md:80]: let's remove this EOL

File: WorkflowGuidelines.md [hunk 2]
```diff
@@ -75,3 +75,80 @@ closed after the commit lands, then you would use the word `Closes` instead of
...
+      ```
+      ```
+      private const int MinimumSupportedDistanceToNotifyKillerDrones = 100;
+
+      ...
+
+      var distance = GpsUtil.GetDistance()
+      if (distance < MinimumSupportedDistanceToNotifyKillerDrones)
+         throw new NotImplementedException();
+      ```
```
Comment (knocte) [inline WorkflowGuidelines.md:97]: let's add an EOL here

File: WorkflowGuidelines.md [hunk 3]
```diff
@@ -75,3 +75,80 @@ closed after the commit lands, then you would use the word `Closes` instead of
...
+             preAuthInput
+             AuthenticationDigestCalculationKey
+
+      ...
+
+      let authInputMac =
+         CalculateMacWithSHA3256
+             authInput
+             AuthenticationDigestCalculationKey
+      ```
```
Comment (knocte) [inline WorkflowGuidelines.md:134]: let's add an EOL here

File: WorkflowGuidelines.md (hunk 2 above)
Comment (aarani) [reply WorkflowGuidelines.md:96]: Done.

File: WorkflowGuidelines.md (hunk 3 above)
Comment (aarani) [reply WorkflowGuidelines.md:134]: Done.

File: WorkflowGuidelines.md (hunk 1 above)
Comment (aarani) [reply WorkflowGuidelines.md]: Done.

Comment (aarani) [reply WorkflowGuidelines.md]: Done.

Comment (aarani) [reply WorkflowGuidelines.md]: Done.

**Pull request merged**